clients/samples/gemm/sample_hipblaslt_gemm_ext_bgradb.cpp

```diff
@@ -0,0 +1,152 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (C) 2024 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+
+#include <hip/hip_runtime.h>
+#include <hipblaslt/hipblaslt-ext.hpp>
+#include <iostream>
+
+#include "helper.h"
+
+void simpleGemmExtEpilogue(hipblasLtHandle_t   handle,
+                           hipblasOperation_t  trans_a,
+                           hipblasOperation_t  trans_b,
+                           hipblasLtEpilogue_t epilogue,
+                           int64_t             m,
+                           int64_t             n,
+                           int64_t             k,
+                           int64_t             batch_count,
+                           float&              alpha,
+                           float&              beta,
+                           void*               d_a,
+                           void*               d_b,
+                           void*               d_c,
+                           void*               d_d,
+                           void*               d_workspace,
+                           void*               d_biasVec,
+                           int64_t             max_workspace_size,
+                           hipStream_t         stream);
+
+int main()
+{
+    /** This is an example using hipblaslt extension API.
+     *  This is a NN example with epilogue HIPBLASLT_EPILOGUE_BGRADB
+     *  A,B,C,D and Bias are all bf16
+     */
+    Runner<hipblasLtBfloat16, hipblasLtBfloat16, hipblasLtBfloat16, float, float> runner(
+        1024, 512, 1024, 1, 1.f, 1.f, 32 * 1024 * 1024);
+
+    runner.setBiasInfo(true, 'B');
+
+    runner.run([&runner] {
+        simpleGemmExtEpilogue(runner.handle,
```

Comment:
For the naming of this function, please refer to "simpleGemmAlphaVecExt". It is better to align the same naming rule even this is just a sample.

Reply:
done, I rename it to simpleGemmEpilogueBiasVecExt()